zetaclient/chains/interfaces/interfaces.go

```diff
@@ -41,13 +41,11 @@ const (
 type ChainObserver interface {
 	Start(ctx context.Context)
 	Stop()
-	VoteOutboundIfConfirmed(
-		ctx context.Context,
-		cctx *crosschaintypes.CrossChainTx,
-	) (bool, error)
+
+	ChainParams() observertypes.ChainParams
 	SetChainParams(observertypes.ChainParams)
-	GetChainParams() observertypes.ChainParams
-	WatchInboundTracker(ctx context.Context) error
+
+	VoteOutboundIfConfirmed(ctx context.Context, cctx *crosschaintypes.CrossChainTx) (bool, error)
```

Comment:
Some comments on each of these functions to explain what they do would be useful here

Reply:
https://github.com/zeta-chain/node/pull/2896/commits/3f8f98958819c72911d6a78660c10490c14e72c6